consul/src/main/scala/io/buoyant/consul/v1/BaseApi.scala

```diff
@@ -49,9 +49,15 @@ trait BaseApi extends Closable {
   private[v1] def mkreq(
     method: http.Method,
     path: String,
+    consistency: Option[ConsistencyMode],
     optParams: (String, Option[String])*
   ): http.Request = {
-    val params = optParams.collect { case (k, Some(v)) => (k, v) }
+    val consistencyMode = consistency match {
```

Comment:
I think `(consistencyMode ++ optParams).collect { case (k, Some(v)) => (k, v) }` should work since both `consistencyMode` and `optParams` are iterables.